catalog-rest-service/src/main/resources/json/schema/entity/tags/tagCategory.json

```diff
@@ -45,15 +42,15 @@
           "description": "Unique name of the tag category.",
           "type": "string"
         },
-        "version" : {
+        "version": {
           "description": "Metadata version of the entity.",
           "$ref": "../../type/entityHistory.json#/definitions/entityVersion"
         },
-        "updatedAt" : {
-          "description": "Last update time corresponding to the new version of the entity in Unix epoch time milliseconds.",
-          "$ref": "../../type/basic.json#/definitions/timestamp"
+        "updatedAt": {
+          "description": "Last update time corresponding to the new version of the entity.",
+          "$ref": "../../type/basic.json#/definitions/dateTime"
         },
```

Comment:
Is this change from timestamp to dateTime intentional?

Reply:
sorry, it seems i made some error merging this. The migration to timestamp was the exact reason i asked in the first place :). reverting ...